packages/webapp/src/components/Animals/SexDetails/SexDetailsCount/index.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import { Text } from '../../../Typography';
+import { Info } from '../../../Typography';
+import { Error } from '../../../Typography';
+import { Details } from '../SexDetailsPopover';
+import SexDetailsCountInput from '../SexDetailsCountInput';
+import styles from './styles.module.scss';
+import { useTranslation } from 'react-i18next';
+
+type SexDetailsCountProps = {
+  maxCount: number;
+  details: Details;
+  total: number;
+  unspecified: number;
+  onCountChange: (countId: Details[0]['id'], count: number) => void;
+};
+
+export default function SexDetailsCount({
+  details,
+  maxCount,
+  total,
+  unspecified,
+  onCountChange,
+}: SexDetailsCountProps) {
+  const { t } = useTranslation();
+  return (
+    <div className={styles.container}>
+      <div className={styles.countInputs}>
+        {details.map(({ id, label, count }) => (
+          <SexDetailsCountInput
+            key={id}
+            max={Math.max(count + unspecified, 0)}
+            label={label}
+            initialCount={count}
+            onCountChange={(count) => onCountChange(id, count)}
+          />
+        ))}
+      </div>
+      <div className={styles.helpText}>
+        <div className={styles.info}>
+          <Info style={{ marginTop: 0 }}>{t('ADD_ANIMAL.ANIMALS_TOTAL', { count: maxCount })}</Info>
+          <Text>{t('ADD_ANIMAL.ANIMALS_UNSPECIFIED', { count: Math.max(unspecified, 0) })}</Text>
```

Comment:
I used the typography components here but they don't match the current designs and I was wondering if we should create the reusable components for various styles of texts?

Reply:
Personally I don't love using the typography components in general, and I might not have utilized them where I should have...

If you find value in using them here, I think you could pass a className to add styles. If not, I would probably mark it with `span`. I wouldn't create reusable components, do you think we should?

Reply:
Reusable text components might be useful in cases where we have the same style of text (like color, font weight), and we can just use the component directly without having to repeatedly create the same classes in css. But I think in this case its fine to just use it with span as you mentioned.